addons/l10n_ro/data/account_tax_data.xml

```diff
@@ -1374,7 +1374,38 @@
     <record id="tvati_extra" model="account.tax.template">
         <field name="chart_template_id" ref="ro_chart_template"/>
         <field name="sequence">35</field>
-        <field name="name">TVA Export</field>
+        <field name="name">TVA Export Bunuri</field>
+        <field name="amount">0</field>
+        <field name="amount_type">percent</field>
+        <field name="type_tax_use">sale</field>
+        <field name="tax_group_id" ref="tax_group_tva_scutit"/>
+        <field name="invoice_repartition_line_ids" eval="[(5,0,0),
+            (0,0, {
+                'factor_percent': 100,
+                'repartition_type': 'base',
+                'plus_report_line_ids': [ref('account_tax_report_ro_baza_rd14')],
+            }),
+            (0,0, {
+                'factor_percent': 100,
+                'repartition_type': 'tax',
+            }),
+        ]"/>
+        <field name="refund_repartition_line_ids" eval="[(5,0,0),
+            (0,0, {
+                'factor_percent': 100,
+                'repartition_type': 'base',
+                'minus_report_line_ids': [ref('account_tax_report_ro_baza_rd14')],
+            }),
+            (0,0, {
+                'factor_percent': 100,
+                'repartition_type': 'tax',
+            }),
+        ]"/>
+    </record>
+    <record id="tvati_extras" model="account.tax.template">
```

Comment:
The id should be unique

Reply:
If I understand correctly your concern that the ids of the 'TVA Export Bunuri' and 'TVA Export Servicii' taxes be different, it seems to me that it's already the case (`tvati_extra` vs `tvati_extras`)

Let me know if I didn't understand correctly.